cmd/kube-apiserver/app/server.go

```diff
@@ -111,6 +111,7 @@ type APIServer struct {
 	SSHUser                    string
 	SSHKeyfile                 string
 	MaxConnectionBytesPerSec   int64
+	ApiServerNodePort          int
```

Comment:
sorry for being dense... how does this interact with `ServiceNodePortRange`? Also, `ApiServer` seems redundant if this is in the APIServer struct (and `Api` should always be written `API` if we end up keeping it in the name)